chat-server/server.js

```diff
@@ -14,13 +22,44 @@ const welcomeMessage = {
 
 //This array is our "data store".
 //We will start with one message in the array.
-//Note: messages will be lost when Glitch restarts our server.
 const messages = [welcomeMessage];
 
-app.get("/", function (request, response) {
+app.get("/", (request, response) => {
   response.sendFile(__dirname + "/index.html");
 });
 
-app.listen(process.env.PORT,() => {
+app.get("/messages", (request, response) => {
+  response.json(messages);
+});
+app.get("/messages/:ID", (request, response) => {
+  const oneMess = request.params.ID;
+  console.log(oneMess);
+  const filterdMessage = messages.find((one) => {
+    console.log(one.id);
+    return one.id == oneMess;
```

Comment:
In general if you're doing a `==` comparison rather than a `===` comparison I'd suggest leaving a comment explaining why so that someone knows it's intentional rather than a typo/bug.